### PR TITLE
refactor: reimplement markdown base component in tailwind

### PR DESCRIPTION
**Pull request by raastinn**
## Pre-Review Checklist
* [x] I have considered backwards and forwards compatibility.
* [x] All changes are tested.
* [x] All touched code documentation is updated.
* [x] All tests pass in my local environment
* [x] Deepsource issues have been reviewed and addressed.
* [x] Commented code and `console` usages have been removed.
 
## Screenshots
_Only required for visual changes_

### CODE REVIEW

File: src/js/base/Markdown.tsx [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-    margin-bottom: 0;
-    padding: 10px 15px;
-`;
-
 type MarkdownProps = {
     markdown?: string;
 };
 
+/**
+ * A styled markdown component
```
Comment (igboyes) [inline src/js/base/Markdown.tsx]: There is much more to this component than that.

**Review by igboyes — CHANGES_REQUESTED** [review]
There is missing work here. Please re-read the issue.

**Comment by raastinn** [comment]
> There is missing work here. Please re-read the issue.

Sorry, didn't read it carefully. Reworked the issue to add it to storybook along with updating the testing and docstrings.

**Review by ReeceHoffmann — APPROVED** [review]

**Pull request merged**